mercury.spec

```diff
@@ -56,7 +59,7 @@ cmake -DMERCURY_USE_CHECKSUMS=OFF                \
       -DMERCURY_USE_SYSTEM_BOOST=ON              \
       -DMERCURY_USE_SELF_FORWARD=ON              \
       -DMERCURY_ENABLE_VERBOSE_ERROR=ON          \
-      -DBUILD_TESTING=ON                         \
+      -DBUILD_TESTING=OFF                        \
```

Comment:
As per suggestion from Jerome building of mercury tests has to be disabled for now due to issues with kwsys